reactor-netty-http/src/main/java/reactor/netty/http/server/logging/AccessLogFactory.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2011-Present VMware, Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package reactor.netty.http.server.logging;
+
+import java.util.function.Function;
+
+/**
+ * An interface to declare more concisely a {@link Function} that apply an {@link AccessLog} by an
+ * {@link AccessLogArgProvider}.
+ * <p>
+ * Can be used in {@link reactor.netty.http.server.HttpServer#accessLog(boolean, AccessLogFactory) accessLog} method
+ * for example.
+ *
+ * @author Simon Baslé
+ * @author Audrey Neveu
+ * @since 1.0.3
+ */
+public interface AccessLogFactory extends Function<AccessLogArgProvider, AccessLog> {
```

Comment:
the plan is still to expose the filtering factories here in the follow-up PR, correct ?

Reply:
Yes, it is.